### ISSUE TITLE
QuestionList answer is not getting piped as question_options and causing error in show_prompts() 

### ISSUE THREAD

**Issue by rbyh** [expectedparrot/edsl#1183]
https://www.expectedparrot.com/content/8fb062ff-2706-4e4c-9343-1c6dc7d063bd

**Comment by rbyh** [comment]
I don't think this is resolved? Issue is gone when entire list answer is used as question options, but not when individual items are referenced:

https://www.expectedparrot.com/content/e149c898-db78-4c8d-aa46-0bae95811ca0

**Comment by rbyh** [comment]
Actually, the list is not getting piped at all now - see last run (showing prompts from the results):

https://www.expectedparrot.com/content/e149c898-db78-4c8d-aa46-0bae95811ca0

**Comment by rbyh** [comment]
Currently these are all happening: 
(1) Unable to pipe a list answer as a (complete) question options list
(2) Unable to pipe individual list answer items 
(3) A survey with a piped list answer will generate an error when show_prompts() is called

**Comment by johnjosephhorton** [comment]
@rbyh Can you check this? I think I fixed all the issues, but they are a bit tricky.

**Comment by rbyh** [comment]
Current errors: https://www.expectedparrot.com/content/b7cf5eb3-9e2e-4a7b-bd96-8ea436066412
